core-library/src/main/java/org/silverpeas/core/admin/service/Admin.java

```diff
@@ -2831,12 +2846,12 @@ public String identify(String sKey, String sSessionId, boolean isAppInMaintenanc
       return getUserSubSpaceIds(componentIds, spaceId);
     } catch (Exception e) {
       throw new AdminException(
-          failureOnGetting("subspaces of space " + spaceId, "accessible by user " + sUserId), e);
+          failureOnGetting(SUBSPACES_OF_SPACE + spaceId, "accessible by user " + sUserId), e);
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "accessible by user " 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://integration.silverpeas.org/sonar/coding_rules#rule_key=squid%3AS1192)